### DISCUSSION TITLE
FAILED state for activated Repositories

### DISCUSSION THREAD

**Discussion by theurichde** [renovatebot/renovate#23114] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Hi there, awesome Renovate folks! 

Today, we activated a bunch of repositories for our organization, all ended up in `FAILED` state, without logs that I could inspect.

Here is a list of failed repos with their job ID:

* 01891ada-4438-72a9-a796-a1980687afbe
* 01891ada-4443-7006-bf99-21d51b8776e9
* 01891ada-443f-7c4b-9e21-137618184668
* 01891ada-4437-73e3-ad7b-4fbf817681c0
* 01891ada-4439-724d-abdd-9bbccfff8204

If you like, I can provide more example IDs. 

One thing I additionally like to notice is, that the Repos weren't visible at all in Renovate until I removed and re-added them to the Renovate app. After that they appeared and I could see the Failed state. We had a drift in the activated Repos vs the Repos available in Renovate (462 visible vs 509 activated)

Thank you!

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Please clarify what you mean by "ended up in failed state"

**Reply by theurichde** [reply]
When you take a look at the job IDs, you can see, that they all have the status "Failed"

```
Status: FAILED
Engine: renovate
Reason: requested
Completed at: 03. Juli 2023, 10:24:22
Duration: -
```

**Reply by rarkins** [reply]
So the _job_ has a failed state, not the repository directly?

Is there any log file for these jobs? If so then please paste one here

**Reply by theurichde** [reply]
The five job IDs above are from five different failed repositories. There are no logs available to look at (for me).

**Reply by svenwilliger** [reply]
as an example:
Job 01891ada-4438-72a9-a796-a1980687afbe

This repo was added yesterday, but there was never an onboarding/configure pull request with the renovate.json file

https://developer.mend.io/[platform]/idealo/seo-upstream-bridge

**Reply by rarkins** [reply]
@theurichde @svenwilliger did this problem resolve itself after some time, or are these repos still "failed"?

**Reply by theurichde** [reply, marked as answer]
Hi @rarkins. No, the problem didn't solve itself. We solved the onboarding by providing `renovate.json`manually on every failed repository. Nonetheless, did the initial onboarding not happen via Renovate, there was no onboarding PR created.

**Reply by theurichde** [reply]
I will come back to you if we encounter this hick-up again on repository level.

**Reply by rarkins** [reply]
Thanks. I'll reopen this discussion because by default GitHub hides closed discussions from user search, and I'd like others users to still find this if they have the same problem
